### PR TITLE
Fixed SimpleSeq2Seq output shape

### PR DESCRIPTION
**Pull request by phdowling**
The output shape of the SimpleSeq2Seq model was previously identical to hidden_dim (if specified). This simply fixes that by changing the last decoder layer's output dimension.

### CODE REVIEW

File: seq2seq/models.py [hunk 1]
```diff
@@ -68,15 +68,24 @@ def SimpleSeq2Seq(output_dim, output_length, hidden_dim=None, depth=1, dropout=0
...
 	decoder.add(Dropout(dropout, batch_input_shape=(shape[0], hidden_dim)))
-	decoder.add(LSTMCell(hidden_dim, **kwargs))
-	for _ in range(1, depth[1]):
-		decoder.add(Dropout(dropout))
+
+	if depth[1] == 1:
+		decoder.add(LSTMCell(output_dim, **kwargs))
+	else:
 		decoder.add(LSTMCell(hidden_dim, **kwargs))
+		for _ in range(1, depth[1] - 1):
```
Comment (farizrahman4u) [inline seq2seq/models.py]: Shouldn't this be depth[1] - 2, since we are adding 2 LSTMCells outside the loop?

Comment (phdowling) [inline seq2seq/models.py]: Well, we also start indexing at 1, so this should be fine right?

Comment (phdowling) [inline seq2seq/models.py]: I mean, we could also replace the whole thing range(depth[1] - 2) for readability

Comment (farizrahman4u) [inline seq2seq/models.py]: Oops, i didn't notice that the range starts from 1. Anyway, either do range(2, depth[1]) or range(depth[1] - 2). Don't mix'em up.

Comment (phdowling) [inline seq2seq/models.py]: Fixed this now.

**Comment by farizrahman4u** [comment]
LGTM, Thanks.

**Pull request merged**